### PR TITLE
Update the personell page with new elections

### PR DESCRIPTION
**Pull request by jaapio**
I'm not sure if I'm supposed to do this, however, this document seemed to be outdated. 

I added 2 years for all people mentioned for re-election. And added me and Navarr to the list of core committee members.

### CODE REVIEW

**Review by Jean85 — CHANGES_REQUESTED** [review]
Thanks for picking this up!

You need to add 2 years to me too though :smile:

**Comment by jaapio** [comment]
Ah sorry, I picked the wrong Alessandro. 

It should now be correct.

File: personnel.md [hunk 1]
```diff
@@ -15,20 +15,20 @@ Feel free to contact the secretaries at info AT php-fig.org. For more informatio
...
 
-| Name                                  | Term                    |
-|---------------------------------------|-------------------------|
-| Larry Garfield ([@Crell])             | 2016-12-24 - 2023-05-30 |
-| Matthew Weier O'Phinney ([@mwop])     | 2016-12-24 - 2023-05-30 |
-| Cees-Jan Kiewiet ([@WyriHaximus])     | 2016-12-24 - 2022-08-31 |
-| Chris Tankersley ([@dragonmantank])   | 2016-12-24 - 2024-01-28 |
-| Korvin Szanto ([@korvinszanto])       | 2016-12-24 - 2024-01-28 |
-| Chuck Burgess ([@ashnazg])            | 2018-08-26 - 2022-08-31 |
-| Woody Gilk ([@shadowhand])            | 2019-05-26 - 2022-08-31 |
```
Comment (shadowhand) [inline personnel.md:26]: 👍🏼

**Review by Jean85 — APPROVED** [review]
Thanks!

**Pull request merged**